### ISSUE TITLE
Run parachain test net collator on our google cloud 

### ISSUE THREAD

**Issue by aliXsed** [NodleCode/chain#627]
Run a parachain test net collator on our google cloud close or internal to the machine where we are running the hot wallet. This may help us investigate the issue that we are observing with our Onfinality nodes plus it can provide a first class service with reduced latency to Hot Wallet

**Comment by simonsso** [comment]
@aliXsed I think the collators should be decoupled from external traffic such as the one generated by the chain-executor.
If we want to provide a service node to hot wallet it should be a fullnode

**Comment by aliXsed** [comment]
Yup @simonsso, agreed. Good point. I'm going to edit this ticket's title and description.
